Sources/SwiftSecurity/Keychain/Keychain.swift

```diff
@@ -233,6 +233,34 @@ extension Keychain: SecItemStore {
         
         return try remove(query)
     }
+
+    public func update<SecItem>(
+        _ value: SecValue<SecItem>,
+        query: SecItemQuery<SecItem>
+    ) throws -> Bool {
```

Comment:
I'd just return Void and let the `notFound` case be handled by the `default` case below. Simpler & more consistent with `store()`